pkg/reconciler/managed/publisher.go

```diff
@@ -27,9 +27,9 @@ type PublisherChain []ConnectionPublisher
 
 // PublishConnection calls each ConnectionPublisher.PublishConnection serially. It returns the first error it
 // encounters, if any.
-func (pc PublisherChain) PublishConnection(ctx context.Context, mg resource.Managed, c ConnectionDetails) error {
+func (pc PublisherChain) PublishConnection(ctx context.Context, owner resource.ConnectionSecretOwner, c ConnectionDetails) error {
```

Comment:
Nit: I think `o resource.ConnectionSecretOwner` or `so resource.ConnectionSecretOwner` would be plenty for these relatively small methods given that the type clearly indicates what the variable is.